client/my-sites/customer-home/main.jsx

```diff
@@ -143,6 +145,29 @@ const Home = ( {
 		return <WooCommerceHomePlaceholder />;
 	}
 
+	const headerActions =
+		config.isEnabled( 'layout/dotcom-nav-redesign' ) &&
+		'wp-admin' === site?.options?.wpcom_admin_interface ? (
+			<>
+				<Button href={ site.URL } onClick={ trackViewSiteAction } target="_blank">
+					{ translate( 'View site' ) }
+				</Button>
+				<Button
+					href={ site.URL + '/wp-admin' }
+					onClick={ trackOpenWPAdminAction }
+					target="_blank"
```

Comment:
I think (for now) we're not supposed to open wp-admin in a new tab (p1707822206412999-slack-C06DN6QQVAQ).

Reply:
Thanks, I updated it.